tests/__fixtures__/settings/configurations.fixtures.ts

```diff
@@ -104,29 +105,28 @@ export const mockConfigurationLoadingState = {
           [mockConfigurationCode]: true,
         },
       },
-      result: [],
+      result: null,
     },
   },
   entities: {
-    configurations: undefined,
+    configurations: {},
   },
 };
 
 export const mockConfigurationsErrorState = {
   settings: {
     configurations: {
-      error: {
-        message: 'An awesome, fascinating and incredible error',
-      },
+      error: null,
```

Comment:
Since this a mock for the error state, shouldn't you have the error instead of `null`?

Reply:
you are right. I've updated it to the correct mock error state. Thanks